### PR TITLE
Add support for aggregated records

### PR DESCRIPTION
**Pull request by jregovic**
Add config option for aggregated records and deaggregation on records in ScanShard

This PR adds an option to consume aggregated records.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,11 +1,12 @@
-module github.com/harlow/kinesis-consumer
+module github.com/jregovic/kinesis-consumer
```
Comment (harlow) [inline go.mod:1]: can we revet this to original

File: consumer.go [hunk 2]
```diff
@@ -78,6 +79,7 @@ type Consumer struct {
 	store                    Store
 	scanInterval             time.Duration
 	maxRecords               int64
+	isaggregated			 bool
```
Comment (harlow) [inline consumer.go:82]: can we `go fmt` this file?

Comment (harlow) [reply consumer.go:82]: also, how about `isAggregated` ?

File: go.mod (hunk 1 above)
Comment (jregovic) [reply go.mod]: Fixed.

File: consumer.go (hunk 2 above)
Comment (jregovic) [reply consumer.go]: Updated both.

**Comment by harlow** [comment]
super cool, thanks for the contribution!

**Pull request merged**